ros2_batch_job/__main__.py

```diff
@@ -507,12 +508,14 @@ def run(args, build_function, blacklisted_package_names=None):
                     'https://github.com/ros2/ros2/releases/download/cryptography-archives/cffi-1.12.3-cp37-cp37dm-win_amd64.whl',  # required by cryptography
                     'https://github.com/ros2/ros2/releases/download/cryptography-archives/cryptography-2.7-cp37-cp37dm-win_amd64.whl',
                     'https://github.com/ros2/ros2/releases/download/lxml-archives/lxml-4.3.2-cp37-cp37dm-win_amd64.whl',
+                    'https://github.com/ros2/ros2/releases/download/untagged-93329b5a3a72a10cef0c/netifaces-0.10.9-cp37-cp37dm-win_amd64.whl',
```

Comment:
`untagged`?

Reply:
@ivanpauno You probably need to tag and cut the release from there.

Reply:
As I commented in the description, I did a draft release.
I can publish the release now before triggering CI, or trigger it and then update the link.

Reply:
You can go ahead and do the release on `ros2` as you always have the ability to re-upload binary artifacts for the release with no impact.

Reply:
Go ahead and trigger a CI build using the branch Michael created. If it passes you can do the tagging / updating of the link.

Reply:
I did the later

Reply:
I finally had to publish the release, because the `untagged` link is not public (you can only see it if you have write access in `ros2/ros2`).
I will retrigger windows debug job when my last commit is pushed in the local branch :smile: 
Can you do that @mjcarroll ?

Reply:
done